redux/auth/types.ts

```diff
@@ -89,6 +90,12 @@ export interface TryAutoLoginAction {
   type: typeof TRY_AUTO_LOGIN;
 }
 
+export interface ClearErrorStateAction {
+  type: typeof CLEAR_ERROR_STATE;
+}
+
+export type errorType = { title: string; message: string };
```

Comment:
Maybe it would be a little better to name this type `"ErrorInfo"` or similar?

Reply:
In my opinion Error itself means that it is some information :thinking: What about just Error? After all, this object is what an error is. What do you think?

Reply:
Okay, but I'm only afraid that it will overlap with JS default [Error](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error) type (it is already defined in TS). Considering this I think my proposal is quite good, but maybe you'll find even better fit :smile:

Reply:
That seems fine :+1: